packages/SwingSet/test/test-gc-actions.js

```diff
@@ -169,7 +170,7 @@ test('gc actions', t => {
   rc = { ko1: [0, 0], ko2: [0, 0] };
   clistState.v1.ko2 = { exists: false, isReachable: false };
   msg = process();
-  t.deepEqual(msg, undefined);
+  t.deepEqual(msg, { type: 'negated-gc-action', vatID: undefined });
```

Comment:
I added this check and it consistently fails.  Some investigation in the debugger calls into question the fundamental correctness of the test's fake `kernelKeeper`.  Since I'm not 100% clear what it *should* be doing, we need to talk.

Reply:
We discussed: the test was not as stateful as I was thinking. The assertion I wanted happens to be handled by the very first clause (labelled "`// idle`"), and does not really need to be exercised at the end. But Chip added an explicit clause for it with a good comment, so I'm completely satisfied.